facebook/FBMock
# [WiP] Adding Composer Support

Just opening up a PR to show progress, will finish it up after my commute. **Do not merge yet**

Done:
- Basic composer.json structure
- Classmap autoloading and Mock.php inclusion

TODO: 
- Test autoloading 
- Add packgist instructions to PR for Facebook crew.
- Verify all configs.
